iron-list.html

```diff
@@ -725,13 +730,15 @@
 
         if (currentPage === 0) {
           // fill the first page
-          this.async(this._increasePool.bind(this, Math.round(this._physicalCount * 0.5)));
+          this._renderDebouncer = this.debounce(this._renderDebouncer,
+              this._increasePool.bind(this, Math.round(this._physicalCount * 0.5)));

```

Comment:
The API for `this.debounce` takes a string as first argument, which is used as a handle for canceling or flushing debouncers.  By passing the object reference, you're effectively using `[Object object]` as the handle, which would collide in the future.

Should be updated to use `debounce(jobName, callback)`, `isDebouncerActive(jobName)` and `flushDebouncer(jobName)`.

The way you are using it is closer to the raw `Polymer.Debounce` library API, rather than the instance API on `Polymer.Base`, fwiw.